File: 0327_Problem_1.ts
// Problem Link: https://leetcode.com/problems/maximum-count-of-positive-integer-and-negative-integer/
// LeetCode Benchmark: 81 ms

function maximumCount(nums: number[]): number {
    return Math.max(
        nums.filter(e => e > 0).length,
        nums.filter(e => e < 0).length)
};

File: 0327_Problem_1.java
// Problem Link: https://leetcode.com/problems/maximum-count-of-positive-integer-and-negative-integer/
// LeetCode Benchmark: 5 ms

public int maximumCount(int[] nums) {
    return Math.max(
        (int) Arrays.stream(nums).filter(e -> e > 0).count(),
        (int) Arrays.stream(nums).filter(e -> e < 0).count());
}
